tardis/montecarlo/montecarlo_numba/single_packet_loop.py

```diff
@@ -31,13 +34,34 @@
 
 C_SPEED_OF_LIGHT = const.c.to("cm/s").value
 
-from tardis.io.logger.montecarlo_logger import log_decorator
-from tardis.io.logger import montecarlo_logger as mc_logger
+# from tardis.io.logger.montecarlo_logger import log_decorator
+# from tardis.io.logger import montecarlo_logger as mc_logger
+
+logger = logging.getLogger("tardis_montecarlo")
+logger.setLevel(logging.DEBUG)
+DEBUG_MODE = True
+
+# logging.basicConfig(
```

Comment:
Commented out some code. This is subject to change based on discussions.

Reply:
Better to just remove if it is not needed. The current montecarlo log does not produce useful information so it may as well be removed. Agree that it should be discussed since some may disagree with me!